internal/ui/login/handler/mfa_verify_u2f_handler.go

```diff
@@ -0,0 +1,53 @@
+package handler
+
+import (
+	"encoding/base64"
+	"net/http"
+
+	http_mw "github.com/caos/zitadel/internal/api/http/middleware"
+	"github.com/caos/zitadel/internal/auth_request/model"
+)
+
+const (
+	tmplU2FVerification = "u2fverification"
+)
+
+func (l *Login) renderU2FVerification(w http.ResponseWriter, r *http.Request, authReq *model.AuthRequest) {
+	userAgentID, _ := http_mw.UserAgentIDFromCtx(r.Context())
+	webAuthNLogin, err := l.authRepo.BeginMfaU2FLogin(setContext(r.Context(), authReq.UserOrgID), authReq.UserID, authReq.ID, userAgentID)
+	if err != nil {
+		l.renderError(w, r, authReq, err)
```

Comment:
instead of render error, render actual form with error? 
also in the following parts where we have render error?